### ISSUE TITLE
Update for 0.9.6/0.13.6

### ISSUE THREAD

**Issue by OoGuru** [dannyti/sboxsetup#9]
How come there has not been an update of installRTorrent for 
rtorrent 0.9.6 http://rtorrent.net/downloads/rtorrent-0.9.6.tar.gz
libtorrent 0.13.6 http://rtorrent.net/downloads/libtorrent-0.13.6.tar.gz
Not much editing needs to be done to implement this.

Regards
Guru

**Comment by dannyti** [comment]
Is it significantly better than 0.9.4 ?

I hear 0.9.4 being banned at few trackers. 

I will update for 0.9.6 if there is greater demand for it and after couple of testing on Debian and Ubuntu latest.

**Comment by dannyti** [comment]
Is it significantly better than 0.9.4 ?

I hear 0.9.4 being banned at few trackers. 

I will update for 0.9.6 if there is greater demand for it and after couple of testing on Debian and Ubuntu latest.

**Comment by OoGuru** [comment]
Okay, I am making some local test with 0.9.6. I just noticed that others had switched to 0.9.6. However, it is hard to follow up on improvements without any changelog because one is forced to read all the commits in the Github, so I cannot tell for sure I must admit.

**Comment by dannyti** [comment]
rtorrent 0.9.6 included in script.

Closed.
